### PR TITLE
Sync files to xxllnc

### PR DESCRIPTION
**Pull request by bbrands02**

### CODE REVIEW

File: Installation/Mapping/XxllncZaakInformatieobjectResultOutMapping.json [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
-  ],
-  "cast": {
-  }
+    "title": "XxllncZaakInformatieobjectBodyOutMapping",
+    "$id": "https://development.zaaksysteem.nl/mapping/xxllnc.XxllncZaakInformatieobjectBodyOutMapping.mapping.json",
+    "$schema": "https://docs.commongateway.nl/schemas/Mapping.schema.json",
+    "version": "0.0.2",
+    "passTrough": true,
+    "mapping": {
+      "#": "{% if results|default %}{{ results|json_encode }}{% endif %}"
```
Comment (WilcoLouwerse) [inline Installation/Mapping/XxllncZaakInformatieobjectResultOutMapping.json:8]: Something to think about, if you haven't already:
What happens here if results is not defined? And does it break stuff if this happens?

File: src/Service/ZGWToXxllncService.php [hunk 2]
```diff
@@ -281,6 +300,7 @@ public function mapZGWToXxllnc(string $casetypeId, ObjectEntity $zaakTypeObject,
      */
     private function getZaakTypeId()
     {
+        return '4a2babc6-0a6d-4ae2-8b27-500cc80423c5';
```
Comment (WilcoLouwerse) [inline src/Service/ZGWToXxllncService.php:303]: You always return the same id here, is this correct?

**Review by WilcoLouwerse — CHANGES_REQUESTED** [review]
Just a small comment, don't forget to add a return type to your functions with ": type" like this:
private function prepareFile(): ?array

File: src/Service/ZGWToXxllncService.php (hunk 2 above)
Comment (bbrands02) [reply src/Service/ZGWToXxllncService.php:303]: Removed, good find

File: Installation/Mapping/XxllncZaakInformatieobjectResultOutMapping.json (hunk 1 above)
Comment (bbrands02) [reply Installation/Mapping/XxllncZaakInformatieobjectResultOutMapping.json:8]: Looks like nothing broke when tested, something broke when the check wasnt there

**Review by WilcoLouwerse — APPROVED** [review]

**Pull request merged**